### ISSUE TITLE
Breaking changes from v2.2.1 to v2.2.2

### ISSUE THREAD

**Issue by jaydenseric** [kamilkisiela/graphql-config#518]
There are undocumented API breaking changes from v2.2.1 to v2.2.2:

https://github.com/kamilkisiela/graphql-config/compare/v2.2.1...v2.2.2

`writeSchema` is no longer exported, and I for one was depending on it.

For example:

https://gist.github.com/jaydenseric/dccd8719ee41741df6d221815a0b0394#file-update-schema-js

The [v2.2.2 release](https://github.com/kamilkisiela/graphql-config/releases/tag/v2.2.2) description “Same as v2.2.1” is frustrating:

<img width="735" alt="Screen Shot 2020-04-17 at 8 33 40 pm" src="https://user-images.githubusercontent.com/1754873/79560357-d277f180-80ea-11ea-8b1a-8bc81a61b09e.png">

Please:

- Consider adding the removed APIs back in a v2 patch release.
- Update the changelog file to reflect the currently undocumented changes.
- Update the GitHub release descriptions.
- Refrain from publishing versions with invalid semver regarding breaking changes moving forwards.

**Comment by kamilkisiela** [comment]
@jaydenseric I took released code from `v2.2.1` and published that. Don't look at the tag on GitHub. Are you sure `v2.2.2` has breaking changes because you checked in code or just on github?

**Comment by jaydenseric** [comment]
> Don't look at the tag on GitHub.

Why would the tag be incorrect? Very confusing.

> Are you sure v2.2.2 has breaking changes because you checked in code or just on github?

I *thought* I had tested it out with a full installation, but maybe I read the changelog for v3 and couldn't see anything about `writeSchema` so I tried it out, got an error, then started walking backwards in the versions tagged in GitHub to see when it was removed. Because the GitHub tag for v2.2.2 doesn't match what was actually published, I thought that was when it was removed.

Just now I created a quick test installation of `graphql-config@2.2.2` and a `writeSchema` function is indeed exported (without testing the actually functionality, but let's assume it works):

```js
const { writeSchema } = require('graphql-config')
console.log(typeof writeSchema)
```

Here are the actual files published to npm:

- https://unpkg.com/browse/graphql-config@2.2.1/lib/index.js
- https://unpkg.com/browse/graphql-config@2.2.2/lib/index.js
- https://unpkg.com/browse/graphql-config@3.0.0/index.cjs.js

In light of this, I apologise for my tone when I said:

> Refrain from publishing versions with invalid semver regarding breaking changes moving forwards.

In my defense I was tired at 8.40 pm on a Friday night, the changelog did not document which version had the breaking change and the GitHub version tag was incorrect.

Actionable:

- Update the changelog file to reflect the currently undocumented changes in v3 (e.g. removal of `writeSchema`).
- Update the GitHub release descriptions to match the changelog updates.
- Correct the GitHub version tags.
